src/Microsoft.SqlTools.ServiceLayer/Security/SecurityService.cs

```diff
@@ -121,7 +121,7 @@ internal async Task HandleCreateLoginRequest(CreateLoginParams parameters, Reque
 
             if (connInfo == null) 
             {
-                    // raise error here
+                throw new ArgumentException("Invalid ConnectionUri");
```

Comment:
in main branch, let's localize the string since it is user facing with some tips about how to recover from this, for example, please reconnect or something like that.

Reply:
I'll create an issue for this in May

Reply:
https://github.com/microsoft/azuredatastudio/issues/22244